fe/src/main/java/org/apache/doris/analysis/FunctionCallExpr.java

```diff
@@ -17,6 +17,11 @@
 
 package org.apache.doris.analysis;
 
+import java.io.DataInput;
```

Comment:
You need to reorder your imports by following:

org.apache.doris
com
org
java
javax